### ISSUE TITLE
Linux compatible?

### ISSUE THREAD

**Issue by GandalftheGreyt3719** [cadon/ARKStatsExtractor#1138]
Im just wondering if this works on linux. im trying to breed my tek paras for farming and im trying to get the event colors but cant tell if its event.

**Comment by cadon** [comment]
Hi, maybe it runs with [mono](https://www.mono-project.com/) or Wine, I haven't tested that personally though.

**Comment by GandalftheGreyt3719** [comment]
ok, i will test

**Comment by alex4401** [comment]
For a bare minimum install, I create a 32-bit prefix, install .NET 4.0 and corefonts with winetricks, then set the version to Windows 7, and then I install .NET 4.7.2 (https://support.microsoft.com/en-us/help/4054530/microsoft-net-framework-4-7-2-offline-installer-for-windows). I'm not exactly sure how well current stable ASB runs through Wine (you might get some exceptions about unavailable speech engine which can be skipped, and an exception when opening settings), but the development version should work fine as far as I've checked.

Summing up:
```bash
$ WINEARCH=win32 wineboot -u
$ WINEARCH=win32 winetricks dotnet40
$ WINEARCH=win32 winetricks win7
$ WINEARCH=win32 wine ~/Downloads/NDP472-KB4054530-x86-x64-AllOS-ENU.exe # or wherever you've downloaded the .NET472 installer
# then install ASB as you normally would, or copy the compiled files into the prefix
```

**Comment by Storm-Engineer** [comment]
> For a bare minimum install, I create a 32-bit prefix, install .NET 4.0 and corefonts with winetricks, then set the version to Windows 7, and then I install .NET 4.7.2 (https://support.microsoft.com/en-us/help/4054530/microsoft-net-framework-4-7-2-offline-installer-for-windows). I'm not exactly sure how well current stable ASB runs through Wine (you might get some exceptions about unavailable speech engine which can be skipped, and an exception when opening settings), but the development version should work fine as far as I've checked.
> 
> Summing up:
> 
> ```shell
> $ WINEARCH=win32 wineboot -u
> $ WINEARCH=win32 winetricks dotnet40
> $ WINEARCH=win32 winetricks win7
> $ WINEARCH=win32 wine ~/Downloads/NDP472-KB4054530-x86-x64-AllOS-ENU.exe # or wherever you've downloaded the .NET472 installer
> # then install ASB as you normally would, or copy the compiled files into the prefix
> ```

Sadly, that didn't work for me, I keep getting an error that .NET 4 is required even though it's installed successfully via winetricks. :(

**Comment by alex4401** [comment]
My instruction is now slightly out of date as ASB has been updated onto .NET 4.8, but I did run some tests and submitted an entry into WINE's AppDB: https://appdb.winehq.org/objectManager.php?sClass=version&iId=39498

The part you'll need is:
> To do this, create a fresh prefix (architecture doesn't seem to matter anymore _**[Wine 5.19 with staging patches at the time]**_) and install dotnet40, dotnet472 and dotnet48 with winetricks.

**Comment by Storm-Engineer** [comment]
> > [Wine 5.19 with staging patches at the time]
I have a problem. I'm playing Overwatch regularly, and it won't work with any Wine version newer than Staging 5.9 so I'm forced to keep it at that version.

**Comment by alex4401** [comment]
I'd recommend having a separate Wine install (personally I manage all prefixes for games through Lutris, incl. Overwatch) in that situation.

**Comment by TheFeshy** [comment]
Just an update for 2024:

I currently use the wine flatpak, and set up a win64 context and a --filesystem that contains the Ark Smart Breeder executable and my saved games.

After using wineboot to create the initial directory structures in the context, I use winetricks to install gdiplus (for me it crashes on some mouse-over without this) and corefonts (without this I do not get displays for gender/cryopod/death/etc.)

I then use winecfg launched from within this context to adjust font scaling, add my game folders to drive locations within wine, and (probably only necessary for some window managers) create a desktop context to give it a better default size.
